core/packages/contracts/contracts/utils/Bitfield.sol

```diff
@@ -37,7 +30,7 @@ library Bitfield {
         uint256[] memory prior,
         uint256 n,
         uint256 length
-    ) public pure returns (uint256[] memory bitfield) {
+    ) internal pure returns (uint256[] memory bitfield) {
 
         // `n` must be <= number of set bits in `prior`
         require(
```

Comment:
So it turns out that this `require` check is redundant, as we already verify the number of set bits in the `prior` bitfield in `submitInitial`. So we can remove it.

Am still keen to incorporate your optimizations to countSetBits, in a separate PR though.

Reply:
Right, because n is the signature threshold which is always <= the 2/3+1 set bits minimum